src/pages/signin/SignInPageLayout/index.js

```diff
@@ -58,26 +58,26 @@ const SignInPageLayout = (props) => {
         </Pressable>
     );
 
-    if (props.isSmallScreenWidth) {
-        return content;
-    }
+    const containerStyles = [];
+    const contentContainerStyles = [styles.flex1];
 
-    if (props.isMediumScreenWidth && props.windowHeight >= variables.minHeightToShowGraphics) {
-        return (
-            <View style={[styles.dFlex, styles.signInPageInner, styles.flexColumnReverse, styles.justifyContentBetween]}>
-                {graphicLayout}
-                <View style={styles.flex1}>
-                    {content}
-                </View>
-            </View>
-        );
+    const isLongMediumScreenWidth = props.isMediumScreenWidth && props.windowHeight >= variables.minHeightToShowGraphics;
+
+    if (props.isSmallScreenWidth) {
+        containerStyles.push(styles.flex1);
+    } else if (isLongMediumScreenWidth) {
+        containerStyles.push(styles.dFlex, styles.signInPageInner, styles.flexColumnReverse, styles.justifyContentBetween);
+    } else {
+        containerStyles.push(styles.flex1, styles.signInPageInner);
+        contentContainerStyles.push(styles.flexRow);
```

Comment:
I think rather than pushing variables into the arrays, via an if/elseif/else it would be clearer to set them directly. So something like: 

```
let containerStyles = [styles.flex1, styles.signInPageInner];
let contentContainerStyles = [styles.flex1, styles.flexRow];

if (props.isSmallScreenWidth) {
    containerStyles = [styles.flex1];
    contentContainerStyles = [styles.flex1];
}

if (isLongMediumScreenWidth) {
    containerStyles = [styles.dFlex, styles.signInPageInner, styles.flexColumnReverse, styles.justifyContentBetween];
    contentContainerStyles = [styles.flex1];
}
```

That way it's very easy to see for each screen size, what the containerStyles and contentContainerStyles are.